addons/sourcemod/scripting/ttt/ttt_inventory.sp

```diff
@@ -0,0 +1,690 @@
+	
+	g_hOnInventoryReady = CreateGlobalForward("TTT_OnInventoryReady", ET_Ignore);
+	g_hOnInventoryMenuItemSelect = CreateGlobalForward("TTT_OnInventoryMenuItemSelect", ET_Ignore, Param_Cell, Param_String);
+	
+	CreateNative("TTT_AddInventoryItem", Native_AddInventoryItem);
+	CreateNative("TTT_RemoveInventoryItem", Native_RemoveInventoryItem);
+	CreateNative("TTT_GetInventoryListing", Native_GetInventoryListing);
+	CreateNative("TTT_IsItemInInventory", Native_IsItemInInventory);
+	
+	RegPluginLibrary("ttt_inventory");
+}
+
+public void OnPluginStart()
+{
+	TTT_IsGameCSGO();
+	LoadTranslations("ttt.phrases");
+	
+	g_cv_EnableCreditLoot = CreateConVar("ttt_enable_loot_credit", "1", "Enables players to loot credits from dead bodies.");
+	g_cv_EnableItemLoot = CreateConVar("ttt_enable_loot_item", "1", "Enables players to loot items from dead bodies.");
+	g_cv_CreditLootMax = CreateConVar("ttt_loot_credit_max", "0", "The amount of credits a player can loot. 0 for unlimited.");
+	g_cv_ItemLootMax = CreateConVar("ttt_loot_item_max", "0", "The amount of items a player can loot per dead body. Duplicates count as an item. 0 for unlimited.");	
+	g_cv_CreditLootChance = CreateConVar("ttt_loot_credit_chance", "0.2", "Chance to loot credits. Default: 0.2 meaning 20%.");
+	g_cv_ItemLootChance = CreateConVar("ttt_loot_item_chance", "0.3", "Chance to loot items. Default: 0.3 meaning 30%.");
+	g_cv_LootChanceFallOff = CreateConVar("ttt_loot_chance_falloff", "0.5", 
+	"Upon successful looting, the ratio at which your next loot chance is reduced by. Default: 0.5 meaning loot chance is reduced by 50%.");
+	
+	RegConsoleCmd("sm_ttt_inventory", Command_Inventory);
+	
+	HookConVarChange(g_cv_EnableCreditLoot, OnConvarChanged);
+	HookConVarChange(g_cv_EnableItemLoot, OnConvarChanged);
+	HookConVarChange(g_cv_CreditLootMax, OnConvarChanged);
+	HookConVarChange(g_cv_ItemLootMax, OnConvarChanged);
+	HookConVarChange(g_cv_CreditLootChance, OnConvarChanged);
+	HookConVarChange(g_cv_ItemLootChance, OnConvarChanged);
+	HookConVarChange(g_cv_LootChanceFallOff, OnConvarChanged);
+	
+	HookEvent("player_death", OnPlayerDeath_Pre, EventHookMode_Pre);
+}
+
+public void OnConfigsExecuted()
+{
+	g_bCreditLoot = GetConVarBool(g_cv_EnableCreditLoot);
+	g_bItemLoot = GetConVarBool(g_cv_EnableItemLoot);
+	g_iCreditLootMax = GetConVarInt(g_cv_CreditLootMax);
+	g_iItemLootMax = GetConVarInt(g_cv_ItemLootMax);
+	g_fCreditLootChance = GetConVarFloat(g_cv_CreditLootChance);
+	g_fItemLootChance = GetConVarFloat(g_cv_ItemLootChance);
+	g_fLootChanceFallOff = GetConVarFloat(g_cv_LootChanceFallOff);
+	
+	g_cv_PluginTag = FindConVar("ttt_plugin_tag");
+	g_cv_PluginTag.AddChangeHook(OnConvarChanged);
+	g_cv_PluginTag.GetString(g_szPluginTag, sizeof(g_szPluginTag));
+}
+
+public Action Command_Inventory(int client, int args)
+{
+	if (!TTT_IsClientValid(client))
+	{
+		return Plugin_Handled;
+	}
+	
+	if (!TTT_IsRoundActive())
+	{
+		ReplyToCommand(client, "%T", "RoundInactive");
```

Comment:
%T need the client index after translation name. It should be: ReplyToCommand(client, "%T", "RoundInactive", client);

Reply:
Oops